File: VB/Dashboard_FirstValueAggregate/Form1.vb
Imports DevExpress.XtraBars.Ribbon

Namespace Dashboard_FirstValueAggregate
	Partial Public Class Form1
		Inherits RibbonForm

		Public Sub New()
			InitializeComponent()
			dashboardDesigner1.CreateRibbon()
			dashboardDesigner1.LoadDashboard("..\..\Data\FirstValueAggregate.xml")
		End Sub
	End Class
End Namespace

File: CS/Dashboard_FirstValueAggregate/Form1.cs
using DevExpress.XtraBars.Ribbon;

namespace Dashboard_FirstValueAggregate {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            dashboardDesigner1.LoadDashboard(@"..\..\Data\FirstValueAggregate.xml");
        }
    }
}
